include/bluetooth/uuid.h

```diff
@@ -30,6 +30,10 @@ enum {
 	BT_UUID_TYPE_128,
 };
 
+#define BT_UUID16_SIZE                  2
```

Comment:
In the spirit of https://github.com/zephyrproject-rtos/zephyr/pull/29375 would you mind adding documentation for these?

Reply:
@joerchan I had a look at #29375, but I do not think I understand what you want.
#29375 deals with endianness in macros.  The defines in this PR are constants, not macros, and they are not going over the air.  I can not see how endianness applies here.

What I can do is something like below - is that what you want?

/** Size in octets of a 16-bit UUID */
#define BT_UUID16_SIZE                  2

Reply:
I meant in the spirit of documenting all macros in the public header.

Yes, that is what I want

Reply:
Fixed.